### DISCUSSION TITLE
Starter Code for Getting Account Details

### DISCUSSION THREAD

**Discussion by mbasith** [hammem/monarchmoney#40] [Q&A] [answered]
I am trying to use this library but failing to get authorized and returned with 401 Unauthorized. I thik I may be using the API incorrectly.  Is there a full boiler plate code that can be provided as an example to successfully get_accounts() information?

**Comment by CalvinChanCan** [comment, marked as answer]
Hi @mbasith.  Here's some boiler plate code I'm using that's been working for me:

```
import asyncio
from monarchmoney import MonarchMoney

async def main():
    mm = MonarchMoney(
        timeout=3000,
    )
    await mm.login(
        email=your_email,
        password=your_password,
        save_session=True,
        use_saved_session=True,
    )
    accounts = await nm.get_accounts()
    



if __name__ == "__main__":
    asyncio.run(main())

```

**Reply by mbasith** [reply]
That worked great, thank you.

**Comment by gregkinney** [comment]
I tried the above code and am geting:
monarchmoney.monarchmoney.RequireMFAException: Multi-Factor Auth Required

I have MFA disabled. Am I now required to use MFA to run this?

**Reply by mattrinella** [reply]
This happens to me as well.  I have MFA disabled on my account and its never been enabled, but the API is saying MFA is required.

monarchmoney.monarchmoney.RequireMFAException: Multi-Factor Auth Required
